src/uct/ib/mlx5/ib_mlx5.h

```diff
@@ -105,6 +105,7 @@ typedef struct uct_ib_mlx5_bf {
         void                    *ptr;
         uintptr_t               addr;
     } reg;
+    unsigned                    size;       /* BlueFlame register size */
```

Comment:
This variable contains size of the register (not the enable/disable flag). In future it might be used in other places.